api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java

```diff
@@ -151,7 +152,19 @@ public DbUser linkRasLoginGovAccount(String authCode, String redirectUrl) {
 
     // If eRA is not already linked, check response from RAS see if RAS contains eRA Linking
```

Comment:
This comment belongs below the ID.me and Login.gov checks - roughly line 168 I'd say.